src/Components/Movies.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import axios from "axios";
+
+export default class Movies extends Component {
+    constructor() {
+        super()
+        this.state = {
+            Gmovies: [], //this represents the array of movies that are objects
```

Comment:
by convention, these state variables should all be lower case - name this `gMovies` instead!

Reply:
I love the comments you left!!